security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc

```diff
@@ -48,10 +48,10 @@ include::modules/cert-manager-acme-dns01-explicit-aws.adoc[leveloffset=+1]
 //Configuring an ACME issuer to solve DNS01 challenges by using ambient credentials on AWS
 include::modules/cert-manager-acme-dns01-ambient-aws.adoc[leveloffset=+1]
 
-// Configuring an ACME issuer to solve DNS01 challenges by using explicit credentials on GCP
+// Configuring an ACME issuer to solve DNS01 challenges by using explicit credentials on {gcp-full}
 include::modules/cert-manager-acme-dns01-explicit-gcp.adoc[leveloffset=+1]
 
-// Configuring an ACME issuer to solve DNS01 challenges by using ambient credentials on GCP
+// Configuring an ACME issuer to solve DNS01 challenges by using ambient credentials on {gcp-full}
```

Comment:
Since attributes are not parsed in comments, this might not be the best fix. Not a blocker. I can handle this one separately. This PR can move on as-is.